### DISCUSSION TITLE
Tenant URL

### DISCUSSION THREAD

**Discussion by edimedia** [fief-dev/fief#371] [Ideas]
The tenant URL is generated like this ` https://default.domain/**tenant_name**` (generated on path), what do you think for this  ` https://**tenant_name**.default.domain/` (on subdomain),

**Comment by fief-bailiff** [comment]
Hail, @edimedia 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to [star our repository](https://github.com/fief-dev/fief) ⭐️

<p align="center">
<img width="656" alt="star-fief" src="https://github.com/fief-dev/fief/assets/1144727/b2db7f59-102f-47e5-9751-67fa10372c52">
</p>

## Want to support us?

Subscribe to one of our paid plan to help us continue our work and receive exclusive information and benefits! Starts at $5/month 🪙

<p align="center">
    <a href="https://polar.sh/fief-dev">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/embed/subscribe.svg?org=fief-dev&darkmode=1">
      <img alt="Subscribe" src="https://polar.sh/embed/subscribe.svg?org=fief-dev">
    </picture>
    </a>
</p>

Farewell!

**Comment by iyobo** [comment]
Subdomains are not very clean.
They also get in the way of domain aliasing, which works best with the current subpath strategy.

As long as your primary domain is different from everything else e.g `auth.mydomain.com`,  the current way is better.
